hoomd/simulation.py

```diff
@@ -109,33 +148,39 @@ def create_state_from_snapshot(self, snapshot):
 
     @property
     def state(self):
+        """State: The current simulation state."""
         return self._state
 
     @property
     def operations(self):
+        """Operations: The operations that apply to the state."""
         return self._operations
 
-    def sanity_check(self):
-        raise NotImplementedError
-
-    def advance(self, runsteps):
-        raise NotImplementedError
-
-    def apply_operations(self, operations):
-        raise NotImplementedError
-
     @property
```

Comment:
Should we make `Simulation` loggable? We could make `TPS` loggable then. Otherwise we can use the `logger['TPS'] = (sim, 'tps', 'scalar')` syntax.

Reply:
Yes, we should. I opened #797 as a reminder and also noted a few other loggable quantities we may want to provide.